### PR TITLE
Add yaml file to /datasets

### PR DESCRIPTION
**Pull request by michaelfaerber**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by michaelfaerber** [comment]
I can't see the error in the yaml file. Any suggestions?

File: datasets/makg.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Description: |
+  The Microsoft Academic Knowledge Graph (MAKG) is a large RDF data set with over eight
+  billion triples with information about scientific publications and related entities, 
+  such as authors, institutions, journals, and fields of study. The data set is 
+  licensed under the Open Data Attributions license.
+Documentation: http://ma-graph.org/
+Contact: http://ma-graph.org/contact/
+ManagedBy: "michael.faerber@cs.uni-freiburg.de"
+UpdateFrequency: Periodically
+Tags:
```
Comment (jflasher) [inline datasets/makg.yaml]: Please add tags as appropriate from https://github.com/awslabs/open-data-registry/blob/master/tags.yaml

File: datasets/makg.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ - aws-pds
+License: http://ma-graph.org/contact/
```
Comment (jflasher) [inline datasets/makg.yaml]: Please link out to the appropriate license.

File: datasets/makg.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+Resources:
+  - Description: nt files
```
Comment (jflasher) [inline datasets/makg.yaml]: Please expand this description a little more.

**Pull request closed without merging**